site/content/en/docs/contrib/building/binaries.md

```diff
@@ -31,9 +31,14 @@ You can also build platform specific executables like below:
     2. `make linux` will build the binary for Linux platform
     3. `make darwin` will build the binary for Darwin/Mac platform
 
-## Compiling minikube using Docker
+## Compiling minikube using 
 
-To cross-compile to/from different operating systems:
+M
+
+```shell
+MINIKUBE_BUILD_IN_DOCKER=y make
```

Comment:
Why use docker? you remove docker from the heading.

Reply:
changed heading